mkdocs.yml

```diff
@@ -635,6 +635,7 @@ nav:
                 - faq/general/how-does-codacy-support-bitbucket-server.md
                 - faq/general/how-does-codacy-keep-my-data-secure.md
                 - faq/general/how-does-codacy-protect-my-privacy.md
+                - faq/general/which-events-does-codacy-log.md
```

Comment:
**question:** Should this have a title that gives more the idea that this feature is not accessible by default to everyone, more like "which events codacy can log"?

Reply:
Thanks, @nfcodacy! Updated. 👍